wesnoth-units.po

```diff
@@ -1607,6 +1623,22 @@ msgid ""
 "offensive swordsmanship and are capable of subverting even the strongest "
 "fortifications."
 msgstr ""
+"Pouštní bojovníci se šavlemi jsou známí tím, že hrají hru zvanou „Laeqad“. V "
+"té na sebe nechají vrhat Laeqads - tlusté a robustní rákosí o délce několika "
+"mečů - a pokouší se je rozseknout po délce během letu ve vzduchu. Trik hry "
```

Comment:
```suggestion
"mečů – a pokouší se je rozseknout po délce během letu ve vzduchu. Trik hry "
```

Reply:
I když to Laeqads by možná ještě zasloužilo nějak vylepšit.